### DISCUSSION TITLE
vcpkg export: Is it safe to merge exported packages together?

### DISCUSSION THREAD

**Discussion by marcbertola** [microsoft/vcpkg#18333] [Q&A]
**Scenario 1**
Alice builds a number of packages for a given triplet:
```vcpkg install --triplet x64-triplet-1 pkg1 pkg2 pkg3```

For whatever reason, Alice decides to export the packages individually:
```vcpkg export --zip --triplet x64-triplet-1  pkg1```
```vcpkg export --zip --triplet x64-triplet-1  pkg2```
```vcpkg export --zip --triplet x64-triplet-1  pkg3```

(I'm assuming here that nothing is changed in the vcpkg repo between the `vcpkg export` calls)

Question 1: Bob wants to use these exported packages. Is it safe for Bob to extract the contents of each zip file and merge them together?
(by "extract", I mean rename the top-level `vcpkg-export-date-time` directory in the .zip archive to a common name for all)

**Scenario 2**
Alice builds a number packages for multiple triplets:
```vcpkg install --triplet x64-triplet-1 pkg1 pkg2 pkg3```
```vcpkg install --triplet x64-triplet-2 pkg1 pkg2 pkg3```

Alice then exports the packages together, per triplet:
```vcpkg export --zip --triplet x64-triplet-1  pkg1  pkg2  pkg3```
```vcpkg export --zip --triplet x64-triplet-2  pkg1  pkg2  pkg3```

Question 2: If Bob wants to work with **both** of these triplets, is it safe to merge the contents exported zip files, as above?

Thanks!

**Comment by lesomnus** [comment]
~Was it safe?~


I merged exported package to vcpkg directory on another machine and no matter to find_package but `vcpkg install something` is not work anymore. It prints `Error: database corrupted.`.
